distribution/sdk/sample-cometd/java-cli-example/src/main/java/org/codice/ddf/cli/modules/GlobalOptions.java

```diff
@@ -33,9 +33,6 @@
     @Option(name = {"-n", "--no-validation"}, title = "Disable Validation", description = "Disables validation of SSL certificates")
     private boolean disableValidation = true;
 
-    @Option(name = "--help", title = "Help", description = "Provides help for the current command")
-    private boolean helpOption = false;
```

Comment:
Thanks for the response  @oconnormi  will do.easy to put back if it gets implemented.